### PR TITLE
feat(entity): simple feature list outside map

### PR DESCRIPTION
**Pull request by LAMM26**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines:https://github.com/infra-geo-ouverte/igo2/blob/master/.github/CONTRIBUTING.md#git-commit-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
Many directories of the MSP have to be transfered to quebec.ca and IGO2 will be used to display various data in these directories. A simplified version of IGO2 has been requested.

**What is the new behavior?**
The user can input a config (SimpleFeatureList) in the compiled version of IGO2 to display a list of the feature contained in the map.

- The user can decide which attributes to show (attributeName), the order of said attributes (attributeOrder) and whether they should have a description or not (description).
- The attributes can also be formatted from other attributes (personalizedFormatting).
- It is possible to sort the elements (sortBy) in the list by a specific attribute value (attributeName).
- It is possible to choose whether URLs and email addresses have to be formatted (Link/Email) or not (google.ca/example@gmail.com) (formatURL and formatEmail)
- A paginator (paginator) can be added and it is possible to specify the number of elements per page (pageSize - default 5) and whether to show First/Previous/Next/Last page buttons or not.
- By selecting an element in the list, the view will update to zoom on said element (partially working)
- By clicking an element in the list, the view will update to zoom on said element and filter the element in the list (partially working).

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications:

**Other information**:
Lib version PR: https://github.com/infra-geo-ouverte/igo2-lib/pull/1062
Docs: Coming soon

### CODE REVIEW

File: src/config/config.json [hunk 1]
```diff
@@ -11,29 +11,38 @@
...
   },
+  "hasFeatureEmphasisOnSelection": false,
+  "menuButtonReverseColor": false,
   "searchSources": {
     "cadastre": {
       "available": false
     }
   },
   "title": "IGO",
-  "welcomeWindow": {
```
Comment (PhilippeLafreniere18) [inline src/config/config.json:32]: keep that config

File: src/config/config.json [hunk 2]
```diff
@@ -11,29 +11,38 @@
...
   "theme": "blue-theme",
-  "hasExpansionPanel": true,
+  "hasExpansionPanel": false,
   "hasGeolocateButton": true,
   "showRotationButtonIfNoRotation": false,
-  "showMenuButton": true,
-  "showSearchBar": true,
-  "hasFeatureEmphasisOnSelection": true,
-  "menuButtonReverseColor": false,
-  "homeExtentButton": {
```
Comment (PhilippeLafreniere18) [inline src/config/config.json:21]: keep that config

File: src/contexts/_default.json [hunk 3]
```diff
@@ -3,45 +3,27 @@
   "base": "_base",
   "layers": [
     {
-      "title": "MSP DESSERTE MUN 911",
+      "title": "WFS",
```
Comment (PhilippeLafreniere18) [inline src/contexts/_default.json:6]: Drag this to another context for the simple-feature-list and keep default as it is

File: src/environments/environment.ts [hunk 4]
```diff
@@ -48,17 +48,6 @@ export const environment: Environment = {
     app: {
       forceCoordsNA: true
     },
-    auth: {
```
Comment (PhilippeLafreniere18) [inline src/environments/environment.ts:51]: keep this config

**Review by PhilippeLafreniere18 — CHANGES_REQUESTED** [review]

File: src/config/config.json (hunk 2 above)
Comment (LAMM26) [reply src/config/config.json:21]: config restored

File: src/config/config.json (hunk 1 above)
Comment (LAMM26) [reply src/config/config.json:32]: config restored

File: src/contexts/_default.json (hunk 3 above)
Comment (LAMM26) [reply src/contexts/_default.json]: Created a new context

**Pull request closed without merging**